packages/react-tracking-pixel/README.md

```diff
@@ -0,0 +1,63 @@
+# `@shopify/react-tracking-pixel`
+
+[![Build Status](https://travis-ci.org/Shopify/quilt.svg?branch=master)](https://travis-ci.org/Shopify/quilt)
+[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](LICENSE.md) [![npm version](https://badge.fury.io/js/%40shopify%2Freact-tracking-pixel.svg)](https://badge.fury.io/js/%40shopify%2Freact-tracking-pixel.svg) [![npm bundle size (minified + gzip)](https://img.shields.io/bundlephobia/minzip/@shopify/react-tracking-pixel.svg)](https://img.shields.io/bundlephobia/minzip/@shopify/react-tracking-pixel.svg)
+
+Allows react apps to easily embed tracking pixel iframes
+
+## Installation
+
+```bash
+$ yarn add @shopify/react-tracking-pixel
+```
+
+## Usage
+
+This library exports a `<TrackingPixel />` component, which allows react apps to easily embed tracking pixel iframes.
+
+### Basic Example
+
+```javascript
+import TrackingPixel from 'react-tracking-pixel';
+
+...
```

Comment:
Eh, invalid code will most likely break `prettier`'s ability to parse + format this as JavaScript.  Turn this line into a comment to work around that.